meson.build

```diff
@@ -16,6 +16,7 @@ endif
 go = find_program('go')
 go_md2man = find_program('go-md2man')
 shellcheck = find_program('shellcheck', required: false)
+codespell = find_program('codespell', required: false)
```

Comment:
Could you please move it above the `shellcheck` line so that the dependencies for the tests stay alphabetically ordered?